tutorials/cli_debugging/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as base
+
+RUN apt update && \
+    apt install -y gdb
```

Comment:
```suggestion
+   apt install -y gdb \
+    ffmpeg 
```
```

Reply:
I needed to add ffmpeg on my x86 workstation, with HSDK 2.2

Reply:
See @zhijinl 's comment -- would you please try again with a clear build cache? Though, I'm fine with adding `ffmpeg` to reduce recurring issues

Reply:
https://github.com/nvidia-holoscan/holohub/pull/433#discussion_r1691581916

Reply:
Added `ffmpeg` with an explanatory comment